.goreleaser.yaml

```diff
@@ -0,0 +1,55 @@
+---
+builds:
+  - id: mailwhale-builds
+    binary: mailwhale
+    dir: .
+before:
+  hooks:
+  - sudo npm install -g yarn
+  - yarn --cwd webui
+  - yarn --cwd webui build
+  - cp config.default.yml config.yml
+archives:
+  - id: mailwhale-archives
+    builds:
+      - mailwhale-builds
+    files:
+      - LICENSE
+      - README.md
+      - webui/public/*
+      - config.default.yml
+      - templates/*
+      - version.txt
+    replacements:
+      darwin: Darwin
+      linux: Linux
+      386: i386
+      amd64: x86_64
+    name_template: "{{ .Binary }}_{{ .Version }}_{{ .Os }}_{{ .Arch }}"
+dockers:
+  - ids:
+      - mailwhale-builds
+    goos: linux
+    goarch: amd64
+    extra_files:
+      - "./go.mod"
+      - "./go.sum"
+      - "main.go"
+      - "version.txt"
+      - "config.yml"
+      - "LICENSE"
+      - "README.md"
+      - "assets/"
+      - "config/"
+      - "service/"
+      - "templates/"
+      - "types/"
+      - "util/"
+      - "web/"
+      - "webui/"
```

Comment:
Unfortunately, this is not allowed. The "intended" way is more or less to just copy the binary that was built by goreleaser, so we would have to add a second "Dockerfile" for the goreleaser builds (in extra_files I would then just have [config.yml, & webui]. I could also move back to the previous approach, by building directly with GitHub Action.

Reply:
I see. Could you perhaps give the approach with a second Dockerfile a try? I think this would also speed up build time, because the binary had already been before and can just be copied then.

Btw., after implementing #41, we wouldn't even need to copy the `webui/` folder anymore. But let's go one step at a time.

Reply:
Updated Readme, copy pasting now allows directly downloading and running ;-)

![image](https://user-images.githubusercontent.com/6696618/214269698-e4672a6e-3379-4e58-9909-15be0ae05800.png)